src/lib/constants.js

```diff
@@ -110,6 +110,8 @@ export const API_ERROR = {
   DEPRECATED_BASIC_HEADER: 'basic authentication is deprecated, please use JWT instead',
   BAD_FORMAT_USER_GROUP: 'user groups is different than an array',
   RESOURCE_UNAVAILABLE: 'resource unavailable',
+  USERNAME_PASSWORD_REQUIRED: 'username and password is required',
+  USERNAME_ALREADY_REGISTERED: 'username is already registered',
```

Comment:
It doesn't make sense. if we are going to have the profile page where the user can change the password and he / she is already logged in ... I do not need to register a new username. only: new password..confirm new password..something similar...and "USERNAME_ALREADY_REGISTERED" only if I have a register page..are we going to have? We need to talk about it.